packages/template-retail-react-app/package.json

```diff
@@ -41,7 +41,6 @@
     "@loadable/component": "^5.15.3",
     "@peculiar/webcrypto": "^1.4.2",
     "@salesforce/commerce-sdk-react": "3.1.0-dev",
-    "@salesforce/pwa-kit-extension-sdk": "4.0.0-dev",
```

Comment:
We don't need the extension-sdk for retail-react-app, is that right?

Reply:
no.. that was probably added by mistake

Reply:
Hmm, CI is failing because of it. So it seems like it's really needed somehow 🤔